gateway/src/apicast/policy/rate_limit/README.md

```diff
@@ -0,0 +1,172 @@
+# Rate Limit policy
+
+- [**Description**](#description)
+- [**Types of limits**](#types-of-limits)
+- [**Define limits**](#define-limits)
+- [**Liquid templating**](#liquid-templating)
+- [**Per-gateway vs shared**](#per-gateway-vs-shared)
+- [**Complete config example**](#complete-config-example)
+
+
+## Description
+
+The purpose of this policy is to control the rate of traffic sent to the
+upstream API. There are several kinds of limits that can be defined. Here are
+some examples:
+
+- The client with address 127.0.0.1 can access the specific endpoint
+`/something` up to 100 times per minute.
+- Requests that contain a JSON web token (JWT) whose field `sub` has the value
+`abc` can make up to 10 calls to the API per second.
+
+
+## Types of limits
+
+The types of limits supported by the policy are the ones provided by the
+[lua-resty-limit-traffic](https://github.com/openresty/lua-resty-limit-traffic)
+library:
+
+- Request: based on the "leaky bucket" method.
+- Count: based on a fixed window of time.
+- Connection: based on the concurrent number of connections.
```

Comment:
We should match the type with the exact type name like `connection_limiters`, `leaky_bucket_limiters` and `fixed_window_limiters`, or add these to the above sentences.